ta/pkcs11/src/pkcs11_attributes.c

```diff
@@ -1243,6 +1255,29 @@ check_parent_attrs_against_processing(enum pkcs11_mechanism_id proc_id,
 
 		return PKCS11_CKR_KEY_FUNCTION_NOT_PERMITTED;
 
+	case PKCS11_CKM_AES_ECB_ENCRYPT_DATA:
+	case PKCS11_CKM_AES_CBC_ENCRYPT_DATA:
+		if (key_class != PKCS11_CKO_SECRET_KEY &&
+		    key_type != PKCS11_CKK_AES) {
+			DMSG("%s invalid key %s/%s", id2str_proc(proc_id),
```

Comment:
i think this trace will add no value. Would you be ok to remove it has done in HMAC cases below?

Reply:
Sure, I will remove the debug trace